dbCreate.sql

```diff
@@ -203,7 +203,7 @@ CREATE TABLE IF NOT EXISTS `Bikes` (
 `BikeType` varchar(128) NOT NULL,
 `NodeId` int(10) unsigned NOT NULL,
 `Position` varchar(128) NOT NULL,
-`Status` boolean NOT NULL,
+`isAvailable` boolean NOT NULL DEFAULT 0,

```

Comment:
I'm not sure about setting default to false here, seems like if we're creating bikes they'd be available by default. Again this isn't really a required change just something to keep in mind when developing the spring app.

Reply:
This was a typo, fixed this in commit 448621db61b9b21c071b28a65a04fe6aaaf3ba86